website/source/docs/providers/aws/r/security_group.html.markdown

```diff
@@ -80,26 +80,24 @@ assign a random, unique name
 
 The `ingress` block supports:
 
-* `cidr_blocks` - (Optional) List of CIDR blocks. Cannot be used with `security_groups`.
+* `cidr_blocks` - (Optional) List of CIDR blocks.
 * `from_port` - (Required) The start port.
 * `protocol` - (Required) The protocol. If you select a protocol of
 "-1", you must specify a "from_port" and "to_port" equal to 0.
 * `security_groups` - (Optional) List of security group Group Names if using
     EC2-Classic or the default VPC, or Group IDs if using a non-default VPC.
-    Cannot be used with `cidr_blocks`.
 * `self` - (Optional) If true, the security group itself will be added as
      a source to this ingress rule.
 * `to_port` - (Required) The end range port.
 
 The `egress` block supports:
 
-* `cidr_blocks` - (Optional) List of CIDR blocks. Cannot be used with `security_groups`.
+* `cidr_blocks` - (Optional) List of CIDR blocks.
 * `from_port` - (Required) The start port.
 * `protocol` - (Required) The protocol. If you select a protocol of
 "-1", you must specify a "from_port" and "to_port" equal to 0.
 * `security_groups` - (Optional) List of security group Group Names if using
     EC2-Classic or the default VPC, or Group IDs if using a non-default VPC.
-    Cannot be used with `cidr_blocks`.
```

Comment:
Worth calling these changes out explicitly; there's nothing in the code that prevents using the two (the docs pre-date the `ConflictsWith[]` helper), and my tests do not reinforce the restriction. I think it's a misinterpretation of the documentation, which says you can't use them, but I believe it's referring to the top level attributes of an `AuthorizeIngress` struct, and not it's `IpPermission` attribute which we use here